.github/workflows/nightly-gauntlet.yaml

```
@@ -3,22 +3,27 @@
 name: nightly-gauntlet
 on:
   schedule:
-    # Every day at midnight
-    - cron: "0 0 * * *"
+    # Every day at 4AM
+    - cron: "0 4 * * *"
```

```suggestion
    - cron: "0 4 * * 1-5"
```
We can skip running on weekends.